File: StringStringBuilderArraylist/removePrime.cpp
#include<iostream>
#include<vector>

using namespace std;

bool isPrime(int n)
    { 
        for(int i = 2 ; i * i <= n ; i++)
        {
            if(n%i==0) return false;
        }
        return true;
    }
void removePrime(vector<int>& al){
		vector<int> ans;
		for(int ele : al){
		    if(!isPrime(ele)) ans.push_back(ele);
		}
		al.clear();
		for(int ele : ans) al.push_back(ele);
	}
int main(int argc, char **argv){
    int n; cin>>n;
    vector<int> arr(n,0);
    for(int i=0; i<n; i++) cin>>arr[i];
    removePrime(arr);
    
    string str = "[";
    for(int ele:arr) str+= to_string(ele) + ", ";
    cout<<str.substr(0,str.length()-2) + "]"; //did -2 to remove the ", " from the end of the array 
        
    return 0;
        
    }

File: StringStringBuilderArraylist/removePrime.java
import java.io.*;
import java.util.*;

public class Main {
    public static boolean isPrime(int n)
    { 
        for(int i = 2 ; i * i <= n ; i++)
        {
            if(n%i==0) return false;
        }
        return true;
    }

	public static void solution(ArrayList<Integer> al){
		ArrayList<Integer> ans = new ArrayList<>();
		for(int ele : al){
		    if(!isPrime(ele)) ans.add(ele);
		}
		al.clear();
		for(int ele : ans) al.add(ele);
	}
	public static void main(String[] args) {
		Scanner scn = new Scanner(System.in);
		int n = scn.nextInt();
		ArrayList<Integer> al = new ArrayList<>();
		for(int i = 0 ; i < n; i++){
			al.add(scn.nextInt());
		}
		solution(al);
		System.out.println(al);
	}

}